seqfu/1.20.3/Dockerfile

```diff
@@ -0,0 +1,71 @@
+# Set global variables
+ARG SEQFU_VER="1.20.3"
+
+# Stage 1: Build Dockerfile
+FROM ubuntu:focal AS builder
+ARG SEQFU_VER
+
+# Metadata
+LABEL base.image="ubuntu:focal" \
+      dockerfile.version="1" \
+      software="SeqFu" \
+      software.version="${SEQFU_VER}" \
+      description="A general-purpose program to manipulate and parse information from FASTA/FASTQ files, supporting gzipped input files." \
+      website="https://github.com/telatin/seqfu2" \
+      license.url="https://github.com/telatin/seqfu2?tab=GPL-3.0-1-ov-file#readme" \
+      maintainer="Taylor K. Paisie" \
+      maintainer.email="ltj8@cdc.gov"
+
+# Set non-interactive frontend and update PATH
+ENV DEBIAN_FRONTEND=noninteractive \
+    PATH="/root/.nimble/bin:${PATH}"
+
+# Install required dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    wget \
+    unzip \
+    build-essential \
+    zlib1g-dev \
+    curl \
+    python3 \
+    python3-pip \
+    git \
+    ca-certificates && \
+    rm -rf /var/lib/apt/lists/*
+
+# Set the working directory
+WORKDIR /root
+
+# Install Nim and SeqFu
+RUN curl https://nim-lang.org/choosenim/init.sh -sSf | sh -s -- -y && \
+    wget https://github.com/telatin/seqfu2/archive/refs/tags/v${SEQFU_VER}.tar.gz && \
+    tar -xzf v${SEQFU_VER}.tar.gz && \
+    rm v${SEQFU_VER}.tar.gz && \
+    cd seqfu2-${SEQFU_VER} && \
+    make && \
+    make test VERBOSE=1 || (echo "Tests failed, but continuing build"; exit 0)
+
+# Copy the built binary to a location in the PATH
+RUN ls -l /root/seqfu2-${SEQFU_VER}/bin && \
+    cp -R /root/seqfu2-${SEQFU_VER}/bin/* /usr/local/bin/ && \
+    chmod +x /usr/local/bin/seqfu
+
+# Stage 2: Create the final image
+FROM ubuntu:focal AS app
+ARG SEQFU_VER
+
+# Copy the necessary files from the builder
+COPY --from=builder /usr/local/bin /usr/local/bin
+COPY --from=builder /root/seqfu2-${SEQFU_VER}/data/tests /root/seqfu2-${SEQFU_VER}/data/tests
```

Comment:
Do you think it's worthwhile to include the test files in the final image? It might be helpful to just have these copied into the test stage for testing if they aren't needed during run time.